coding-concepts.js

```diff
@@ -7,62 +7,62 @@
 
 // --------------------1) What will this log?
 
-const cohort = "Golf 2022"
-// console.log(cohort.split(""))
+const cohort = "Golf 2022";
+//console.log(cohort.split(""));
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:the console.log should split the strings value with a "" seperating each letter.
+// b) Verify and explain: the letters were seperated with ''.
+//.split() will take the contents of an string and return them seperated in a new array.
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
-  `Hello, ${name}!`
-}
-// console.log(greeter("LEARN Student"))
+  `Hello, ${name}!`;
+};
+//console.log(greeter("LEARN Student"));
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: it will return what ever value we input as "name" and return the comment `Hello, LEARN Student!`
+// b) Verify and explain: came back with undefined, because the function was missing its return command. i put return at the beginning of line 20 and the console.log returned the expected output.
 
 // --------------------3) What will this log?
 
-const multipliedByTwo = [4, 5, 6, 7, 8].map((number) => number * 2)
-// console.log(multipliedByTwo)
+const multipliedByTwo = [4, 5, 6, 7, 8].map((number) => number * 2);
+//console.log(multipliedByTwo);
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: the function is missing a return. i expect the log to return as undefined.
+// b) Verify and explain: the log worked and returned the numbers in the array multiplied by 2. i think this is because the .map will return the values of the array in a new array. not needing a return.
```

Comment:
multipliedByTwo in this case is technically just a variable that is assigned to the new array that map will provide.  Inside the map, we are using implicit return because we are returning on the same line.